syftr/__init__.py

```diff
@@ -1,3 +1,8 @@
-from importlib.metadata import version as _version
+try:
+    from importlib.metadata import version
 
-__version__ = _version("syftr")
+    __version__ = version("syftr")
+except Exception:
+    import os
+
+    __version__ = os.getenv("SYFTR_VERSION", "0.0.0")
```

Comment:
The logic here is confusing enough that it warrants some good module docstring explaining the different cases for setting the version

Reply:
added some notes